src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.UnitTestingAsyncWorkItemQueue.cs

```diff
@@ -259,6 +273,7 @@ protected CancellationToken GetNewCancellationToken_NoLock(object key)
                             return projectId;
                         }
 
+#if false // Not used in unit testing crawling
                         // prefer project that directly depends on the given project and has diagnostics as next project to
                         // process
```

Comment:
irrelevant to unittesting solution crawling.  source-discovery is unrelated to where diagnostics are.